README.md

```diff
@@ -13,9 +13,9 @@ applications' configurations, leveraging easy application integration.
 
 ## Often used links ##
 
-- [build server](http://build.libelektra.org:8080/)
-- [tutorials](http://git.libelektra.org/blob/master/doc/tutorials/)
-- [API docu](http://doc.libelektra.org/api/latest/html/)
+- [build server](https://build.libelektra.org/)
```

Comment:
ah yeah i've already fixed that but haven't comitted sorry, now it should be in the branch